docs/docs/09-troubleshooting.md

```diff
@@ -23,4 +23,34 @@ module.exports = {
 };
 ```
 
+### No such file or directory
 
+If you encounter a warning message such as this:
+
+```
+ENOENT: no such file or directory, open …/node_modules/csstype/index.js
+```
```

Comment:
This was the error reported. While we should probably raise a more specific error for this instance in Snowpack, this is a quick fix we can release now.